extensions/interactions/LogicProof/directives/logic_proof_response_directive.html

```diff
@@ -1,5 +1,5 @@
 <div>
-  <[answer.displayed_question]>
+  <p>Assuming <[answer.assumptions_string]>; prove <[answer.target_string]>.</p>
```

Comment:
Can we use `I18N_INTERACTIONS_LOGIC_PROOF_QUESTION_STR_ASSUMPTIONS` here?

Reply:
I switched to using `I18N_INTERACTIONS_LOGIC_PROOF_QUESTION_STR_ASSUMPTIONS`